app/controllers/saml_idp_controller.rb

```diff
@@ -56,7 +56,7 @@ def render_template_for(message, action_url, type)
   end
 
   def delete_branded_experience
-    session.delete(:sp) if session[:sp]
-    session.delete(:user_return_to) if session[:user_return_to]
+    session.delete(:sp)
+    session.delete('user_return_to')
```

Comment:
Should we change all our other custom session keys to be strings to be consistent?